Digital Clock/PageSetAlarm.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick
+
+Rectangle{
+    id: root
+    width: 480
+    height: 800
+    color: "#151B2E"
+    Image {
+        id: goBack
+        source: "/assets/btn-back-hover.svg"
+        anchors.left: root.left
+        anchors.top: root.top
+        anchors.leftMargin: 28
+        anchors.topMargin: 20
+    }
+
+    Text {
+        id: alClockList
+        text: qsTr("Alarm")
+        color: "#9FAAB0"
+        font.pixelSize: 30
+        anchors.horizontalCenter: root.horizontalCenter
+        anchors.top: root.top
+        anchors.topMargin: 40
+    }
+
+    Rectangle{
+        width: 424
+        height: 1
+        color: "#9FAAB0"
+        anchors.horizontalCenter: root.horizontalCenter
+        anchors.top: root.top
+        anchors.topMargin: 120
+    }
+
+    Button3{
+        id: everyday
+        testo: "Everyday"
+        state: "hover"
+        anchors.top: root.top
+        anchors.topMargin: 140
+        anchors.left: root.left
+        anchors.leftMargin: 34
+
+        MouseArea{
+            anchors.fill: parent
+            onClicked: {
+                if(parent.state === "hover"){
+                    if(date.state === "active"){
+                        date.state = "hover"
+                        date.testo = "Set date"
+                    }
+                    parent.state = "active"
+                }
+                else{
+                    parent.state = "hover"
+                }
```

Comment:
Quando le cose sono semplici è sempre meglio no? Qui si stanno complicando un pochino perché stai facendo gestire ai singoli pulsanti il fatto che solo uno può essere attivo contemporaneamente.
Ti torna che se mettevi una property nel componente in cima, e mettevi lo stato dei pulsanti in binding su quella property, per esempio `state: root.isEveryday ? "active" : "hover"` e al contrario sull'altro, ti basta cambiare la property?
`onClicked: root.isEveryday = true` (o qualcosa del genere).